Libraries/PeriphDrivers/Source/RTC/rtc_me30.c

```diff
@@ -65,6 +65,30 @@ int MXC_RTC_SetSubsecondAlarm(uint32_t rssa)
     return MXC_RTC_RevA_SetSubsecondAlarm((mxc_rtc_reva_regs_t *)MXC_RTC, rssa);
 }
 
+int MXC_RTC_SetClockSource(mxc_rtc_clock_t clk_src)
+{
+    uint8_t retval = E_NO_ERROR;
+
+    switch (clk_src) {
+    case MXC_RTC_ERTCO_CLK:
+        MXC_GCR->clkctrl |= MXC_F_GCR_CLKCTRL_ERTCO_EN;
```

Comment:
Please use "MSDK_NO_GPIO_CLK_INIT" macro in here.

Reply:
Done, thanks